tool/gravity/cli/status.go

```diff
@@ -75,7 +76,9 @@ func status(env *localenv.LocalEnvironment, printOptions printOptions) error {
 		}
 	}
 	if status.Agent == nil {
-		status.Agent, err = statusapi.FromPlanetAgent(context.TODO(), nil)
+		ctx, cancel := context.WithTimeout(context.TODO(), defaults.StatusCollectionTimeout)
```

Comment:
By Re-creating the context, is the intention that when falling back to give an additional 1-minute for collecting status to complete?

Reply:
Yeah, that's what I was thinking.